DataConnectors/Perimeter81ActivityLogs.json

```diff
@@ -0,0 +1,85 @@
+{
+  "id": "Perimeter81ActivityLogs",
+  "title": "Perimeter 81 Activity Logs",
+  "publisher": "Perimeter 81",
+  "descriptionMarkdown": "The Perimeter 81 Activity Logs connector allows you to easily connect your Perimeter 81 activity logs with Azure Sentinel, to view dashboards, create custom alerts, and improve investigation.",
+  "graphQueries": [
+    {
+      "metricName": "Total data received",
+      "legend": "Perimeter81_CL",
+      "baseQuery": "Perimeter81_CL"
+    }
+  ],
+  "sampleQueries": [
+    {
+      "description" : "User login failures",
+      "query": "Perimeter81_CL | where eventName_s == \"api.activity.login.fail\""
+    },
+    {
+      "description" : "Application authorization failures",
+      "query": "Perimeter81_CL | where eventName_s == \"api.activity.application.auth.fail\""
+    },
+    {
+      "description": "Application session start",
+      "query": "Perimeter81_CL | where eventName_s == \"api.activity.application.session.start\""
+    }
+  ],
+  "dataTypes": [
+    {
+      "name": "Perimeter81_CL",
+      "lastDataReceivedQuery": "Perimeter81_CL\n | summarize Time = max(TimeGenerated)\n | where isnotempty(Time)"
+    }
+  ],
+  "connectivityCriterias": [
+    {
+      "type": "IsConnectedQuery",
+      "value": [
+        "Perimeter81_CL\n | summarize LastLogReceived = max(TimeGenerated)\n | project IsConnected = LastLogReceived > ago(30d)"
+      ]
+    }
+  ],
+  "availability": {
+    "status": 1
+  },
+  "permissions": {
+    "resourceProvider": [
+      {
+        "provider": "Microsoft.OperationalInsights/workspaces",
+        "permissionsDisplayText": "read and write permissions are required.",
+        "providerDisplayName": "Workspace",
+        "scope": "Workspace",
+        "requiredPermissions": {
+          "write": true,
+          "read": true,
+          "delete": true
+        }
+      }
+    ]
+  },
+  "instructionSteps": [
+    {
+      "title": "",
+      "description": "Please note the values below and follow the instructions <a href='https://support.perimeter81.com/hc/en-us/articles/360012680780'>here</a> to connect your Perimeter 81 activity logs with Azure Sentinel.",
```

Comment:
Really like the very thorough step-by-step guidance you have on the onboarding side. Thanks, for the effort.